migrations/1578929548449-add bbb template to lti tools.js

```diff
@@ -0,0 +1,66 @@
+const mongoose = require('mongoose');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to resolve path to module 'mongoose'.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4819756)